tutorials/detecting-changes-in-sentinel-1-imagery-pt-4/Tutorial_pt_4.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<ul><li>If you have open, reputable articles for the specific events mentioned, please link to them, the tutorial will age better.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/422/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I've added three links concerning turaround time statistics, Benghasi port closure and coal mine decommissioning.</p>

Reply:
<p>Justin to change the Port of Benghazi article link to the Reuters source quote</p><p>https://www.reuters.com/article/libya-security-benghazi-port-idINKCN1C61MC </p>